onnx/test/automatic_upgrade_test.py

```diff
@@ -1,1813 +1,1821 @@
-# Copyright (c) ONNX Project Contributors
```

Comment:
EOL is wrong.

Reply:
there is an easy way in VS CODE to change EOL. Done